tools/depends/target/libnfs/Makefile

```diff
@@ -9,7 +9,7 @@ COMMITDATE=2020-06-09
 ARCHIVE=$(LIBNAME)-$(VERSION).tar.gz
 
 # configuration settings
-CONFIGURE=./configure --prefix=$(PREFIX) --disable-shared --disable-utils --disable-examples
+CONFIGURE=./configure --prefix=$(PREFIX) --disable-shared --disable-utils --disable-examples --disable-werror
```

Comment:
This seems odd. There must be a better solution? Seems like you might just need `-Wno-builtin-macro-redefined` ?

Reply:
realistically does it matter either way? We use it, we dont dev it. what do we care about warnings in libnfs at dependency build time?

Reply:
and btw, this is a configure switch in libnfs, not a build flag